documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex

```diff
@@ -226,10 +228,97 @@ \subsubsection{Load increase mode: simulation of load increase}
+
+\lstinputlisting[language=XML, morekeywords={id},title=multiple jobs file example]{../resources/syntaxExample/CTCSyntaxExample.xml}
+
+In this example, the basic initial situation is described in the \textbf{IEEE14.jobs} file. The format used is the same as a \Dynawo simulation.
+
+Three scenarios will be applied: `NodeFault1', `NodeFault2' and `NodeFault3'. Each event is described in an additional \textbf{dyd} file, which will be added individually to the main \textbf{dyd} file described in the \textbf{jobs} file.
+Below is an example of a \textbf{dyd} file for an event:
+
+\lstinputlisting[language=XML, morekeywords={id},title=event dyd file example]{../resources/syntaxExample/CTCEvent.dyd}
+
+The critical time calculation requires certain parameters to perform its dichotomy.
+\begin{itemize}
+  \item accuracy: accuracy of the calculation;
+  \item dydId: ID of the element containing the fault in the \textbf{dyd} file;
+  \item parName: name of the double that handles the fault end in the \textbf{par} file;
+  \item minValue: min time limit for the dichotomy calculation;
+  \item maxValue: max time limit for the dichotomy calculation;
+  \item mode: [optional, default= `SIMPLE'] dichotomy calculation mode.
+\end{itemize}
+
+The mode is optional and can be either `SIMPLE' or `COMPLEX'. `SIMPLE' mode performs a classic dichotomy. If solver problems (linearity issue) occur when running this mode, this can lead to errors in the calculation of the critical time. In this case, the user can restart the calculation by switching the mode to `COMPLEX'. When the algorithm encounters one of these problems with tmax, it returns to the last value tested that caused the simulation to diverge normally and removes the accuracy once. It then resumes the dichotomy but it may lengthen the calculation.
+
+This \textbf{dyd} is added to the one given in \textbf{IEEE14.jobs} and adds a node fault to `FAULT\_GEN\_1', which is then connected to `\_BUS\_\_\_\_1\_TN\_ACPIN'.\\
+
+The command line required to launch CTC mode is the following:
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh CTC --directory <PATH TO WORKING DIRECTORY> --input <NAME OF INPUT FILE> --output <NAME OF OUTPUT FILE> --nbThreads <NUMBER OF THREADS>
+\end{lstlisting}
+
+The default number of thread is 1. The input can be explicit files or zipped ones. For example:
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh CTC --directory nrt/data/IEEE14/CTC --input fic_MULTIPLE.xml --output results.xml --nbThreads 3
+\end{lstlisting}
+
+This command will launch the critical time calculation described in the file \textit{nrt/data/IEEE14/CTC/fic\_MULTIPLE.xml} in multi-threading 
+and provides as outputs a file named \textit{results.xml} containing the status of all scenarios that were executed, 
+in addition to the last logs and requested outputs of the last simulation of each scenario.
```

Comment:
Not clear to me what does "last" mean here? The very last simulation, whether it passed or not?